selftests/check.py

```diff
@@ -518,6 +519,19 @@ def create_suites(args):
     return suites
 
 
+def print_failed_tests(job_log_dir):
+    jfile = os.path.join(job_log_dir, '.journal.sqlite')
```

Comment:
Interesting that you found a use for the journal, and at the same time, it's a kind of extra check *for* the journal.